Resources/Locale/en-US/loadouts/jobs/heads/chiefMedicalOfficer.ftl

```diff
@@ -0,0 +1,5 @@
+loadout-description-LoadoutCommandCMONeckMantle = To show who has the authority around here. It seems unusually clean.
+loadout-description-LoadoutCommandCMONeckCloak = To really show who has the authority around here. It seems unusually clean.
+loadout-description-LoadoutCommandCMOOuterWinter = A warm coat for the cold of space. It seems unusually clean.
+loadout-description-LoadoutCommandCMOOuterLab = A lab coat for the CMO. It seems unusually clean.
+loadout-description-LoadoutCommandCMOHatBeret = A beret for the CMO. It seems unusually clean.
```

Comment:
English localisation for entity prototypes should not be committed to the repo.

Reply:
These are loadout prototypes, not entities